app/components/ScrollToButton.tsx

```diff
@@ -20,9 +21,20 @@ export function useScrollToEvent(scheduleIndex) {
   const currentEventIndex = useSharedValue(-1)
   const currentlyViewingEvents = useSharedValue<number[]>([])
   const currentlyViewingSchedule = useSharedValue(0)
+  const isFocused = useSharedValue(false)
+
+  useFocusEffect(
+    useCallback(() => {
+      isFocused.value = true
+      return () => {
+        isFocused.value = false
+      }
+    }, []),
+  )
```

Comment:
Even though it was firing `onViewableItemsChanged` on a different screen which was causing the issue, I still needed `itemVisiblePercentThreshhold` for the button to show up after passing the current event card, since Frank and I found out that the `isFocused` is still getting called when the screen changes, adding another shared value like this fixed the issue.

Reply:
Ah I see you mention shared value explicitly here, does it not with with a plain value either useState or useIsFocused?

Curious!